index.html

```diff
@@ -0,0 +1,289 @@
+                    <ul>
+                        <li><div id="work4tag1"><div id="html4">html</div></div></li>
+                        <li><div id="work4tag2"><div id="css4">css</div></div></li>
+                        <li><div id="work4tag3"><div id="javascript4">javascript</div></div></li>
+                    </ul>
+                </div>
+                <div class="work4Actions">
+                    <button id="work4see-project">
+                       <b>See Project</b>
+                    </button>
+                </div>
+               </div>
+               
+            </div>
+        </div>
+        <section class="About-myself">
+            <section class="top">
+                <div id="primary-heading"><h2>About Myself</h2></div>
+                <div id="primary-paragraph">
+                    <p>Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    </div>
+                    <div class="lets-connect">LET’S CONNECT</div>
+                    <div class="connect-icons">
+                        <ul>
+                            <li><img src="Vector2.png" alt=""></li>
+                            <li><img src="Linkedin.png" alt=""></li>
+                            <li><img src="Vector3.png" alt=""></li>
+                            <li><img src="Vector.png" alt=""></li>
+                            <li><img src="Vector1.png" alt=""></li>
+                        </ul>
+                    </div>
+                     <button id="my-resume"><span>Get my resume</span></button>
+            </section>
+            <div class="skill-list">
+                    <div id="frame12">
+                        <div id="language">
+                            <div id="drop-down">
+                                <img src="Chevron-Down.png" alt=""></div>
+                                <div id="lang">
+                                    <h5>Languages</h5>
+                                </div>
+                            
+                        </div>
+                    </div>
+                    <div id="lang-list">
+                        <div id="frame9">
+                            <div id="img"><img src="Ellipse1.png" alt=""></div>
+                            <div id="js">javascript</div>
+                            
+                        </div>
+                    </div>
+                    <div id="frame10">
+                        <div id="img1"><img src="Ellipse2.png" alt=""></div>
+                            <div id="html">HTML</div>
+                    </div>
+                    <div id="frame11">
+                        <div id="img2"><img src="Ellipse0.png" alt=""></div>
+                            <div id="css">CSS</div>
+                    </div>
```

Comment:
- [ ] I noticed the same thing here, Images not working as expected I think the path is wrong (src=" "), kindly fix it. 👌

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/177231439-6d64eaea-c560-46c0-b0b7-f0923038158a.PNG" width="100%" />  | <img src="https://user-images.githubusercontent.com/45231276/177231491-929b1eef-a842-4815-a5c9-f23f910060c2.png"/> |